.github/workflows/docker.yml

```
@@ -10,16 +10,16 @@ jobs:
   build_docker:
     strategy:
       matrix:
-        rust-features: [["default"], ["integration-testing" , admin"]]
+        rust-features: [["default"], ["integration-testing", admin"]]
```

The issue is a missing quote here. I guess the matrix values are all string-typed, so the unbalanced quotes just get substituted into bash.
```suggestion
        rust-features: [["default"], ["integration-testing", "admin"]]
```

---

I was also failing to access `matrix.images.rust-features` on the `docker-release` side, fixed now.